macros/form.njk

```diff
@@ -43,6 +43,28 @@
   </div>
 {% endmacro %}
 
+{% macro searchInput(label, name, form, hint, bold = false) %}
+  {% set error = form.errorFor(name) %}
+  <div class="form-group {% if error %} form-group-error {% endif %}">
```

Comment:
you're adding a space here always when there's no error, tbh lots of extra spaces 😉